modules/microshift-config-yaml.adoc

```diff
@@ -23,24 +23,36 @@ network:
   serviceNetwork:
     - 10.43.0.0/16 <3>
   serviceNodePortRange: 30000-32767 <4>
+  advertiseAddress:
```

Comment:
This flag is under the `apiServer` field. It is a string instead of a list. It's default value is `10.44.0.0/32`. 

```
apiServer:
  advertiseAddress: 10.44.0.0
```